src/components/CompletedSubReqCourse.vue

```diff
@@ -126,6 +129,9 @@ export default Vue.extend({
       this.courseSubject = crseTaken.subject;
       this.courseNumber = crseTaken.number;
     },
+    onReset() {
+      this.$emit('deleteCourseFromSemesters', this.courseSubject, this.courseSubject);
+    },
```

Comment:
Reset functionality doesn't seem to work because you're missing uniqueID. I think this would involve creating a new data field like `courseUniqueID` and then setting `this.courseUniqueID to be the `uniqueID` of the course that is found in `setCourseAndSemesterForNonTransferCredits()`.

Reply:
Lol I didn't check off the file on GitHub desktop. Sorry about that. Should be pushed now